vendor.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"go/parser"
+	"go/token"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+func main() {
+
+	wd, err := os.Getwd()
+	paniconerr(err, "getting working dir")
```

Comment:
instead of panic, why not log the error on stderr and exit with status != 0 ?
There's no 'panic' here, you can handle the error better.

Reply:
I changed to panic because I wanted to guarantee the defers are executed. Just calling os.Exit may cause the tempdir to be left on disk. Besides that...there is no reason =)